README.md

```diff
@@ -245,6 +245,10 @@ Clone code and cd to directory
     git clone git@github.com:awslabs/llrt.git --recursive
     cd llrt
 
+Install git submodules if you've not cloned the repository with `--recursive`
```

Comment:
Maybe a bit redundant given the notice below? Also first instruction states that you should clone with recurisve